### ISSUE TITLE
Waiting input on Travis because duplicated files on shards install

### ISSUE THREAD

**Issue by faustinoaq** [amberframework/amber#1047]
### Description

I noticed some PR are failing on Travis, I think is because `shards` command is asking for action on duplicated files:

```
Installing selenium (0.4.0)
........spec/controllers/spec_helper.cr already exists...
overwrite(o)/keep(k)/diff(d)/overwrite all(a)/keep all(n)/quit(q) ? 
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated
```

See: https://travis-ci.org/amberframework/amber/jobs/474006019#L801

### Steps to Reproduce

See failing PR on https://github.com/amberframework/amber/pulls

### Versions

Master (https://github.com/amberframework/amber/commit/39460a5a46d38efc44235e9856e8ecd0e0d411a9)

### Additional Information

Should we try something like `yes a | shards`, WDYT?

@amberframework/contributors

**Comment by nsuchy** [comment]
Can this issue still be reproduced?

**Comment by drujensen** [comment]
This is fixed in #1018
